### ISSUE TITLE
sls deploy with --optional_parameter doesn't work anymore

### ISSUE THREAD

**Issue by yeyeric** [serverless/serverless#8950]
Hello,

In my cicd (gitlab), using the latest framework available of serverless, since 1h, the following bug occurred (nothing changed in my cicd for 2 weeks so can't be due to a wrongdoing with it):

`- serverless deploy --stage $CI_COMMIT_REF_NAME --url $CI_ENVIRONMENT_URL --bucket_data_name $BUCKET_DATA_NAME_FINAL`

The following command bugs at --bucket_data_name :   "--BucketDataName" is not a valid sub command. Run "serverless deploy" to see a more helpful error message for this command.

How do we pass optional parameter when deploying ?

<b>Installed version: </b>

```
2.24.0
```

**Comment by medikoo** [comment]
@yeyeric thanks for report. Can you post what exactly error message you're getting? Paste exact output as it's in a CLI?

**Comment by yeyeric** [comment]
serverless deploy --stage $CI_COMMIT_REF_NAME --url $CI_ENVIRONMENT_URL --bucket_data_name $BUCKET_DATA_NAME_FINAL
 
  Serverless Error ---------------------------------------
 
  "--bucket_data_name" is not a valid sub command. Run "serverless deploy" to see a more helpful error message for this command.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              14.15.5
     Framework Version:         2.24.0
     Plugin Version:            4.4.3
     SDK Version:               2.3.2
     Components Version:        3.7.0

**Comment by yeyeric** [comment]
@medikoo 
let me know if that's good:

serverless deploy --stage $CI_COMMIT_REF_NAME --url $CI_ENVIRONMENT_URL --bucket_data_name $BUCKET_DATA_NAME_FINAL
 
  Serverless Error ---------------------------------------
 
  "--bucket_data_name" is not a valid sub command. Run "serverless deploy" to see a more helpful error message for this command.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              14.15.5
     Framework Version:         2.24.0
     Plugin Version:            4.4.3
     SDK Version:               2.3.2
     Components Version:        3.7.0

**Comment by medikoo** [comment]
Thank you that's great, fix is coming

**Comment by ankon** [comment]
This also broke our automatic deployments, as a workaround I now have a `"serverless": ">=2.16.1 <2.24.0"` in my package.json (and fixed an issue in my pipeline to use package locks when installing!)

**Comment by medikoo** [comment]
Sorry about issues it caused, fix was published with v2.25.0

**Comment by wviana** [comment]
I still having the same problem on 2.25, here is my output

```
  Serverless Error ---------------------------------------
 
  "--ARN_S3_BATCH_IMPORT_BITRIX" is not a valid sub command. Run "serverless deploy" to see a more helpful error message for this command.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              14.4.0
     Framework Version:         2.25.0 (standalone)
     Plugin Version:            4.4.3
     SDK Version:               2.3.2
     Components Version:        3.7.0
```

**Comment by medikoo** [comment]
@wviana great thanks for reporting. Fixed and published with v2.25.1
